README.md

```diff
@@ -38,7 +38,7 @@ The output would be:
 applyFilter({
   where: {
     // the price > 10 && price < 100
-    price: {gt: 10, lt: 100},
+    and: [{id: {gt:10}}, {id:{lt:100}}],
```

Comment:
any reason we change from price to id? 
from the comment, it's also about `price`.